client/src/commonMain/kotlin/com/algolia/search/endpoint/internal/EndpointPersonalization.kt

```diff
@@ -0,0 +1,73 @@
+@file:Suppress("FunctionName")
+
+package com.algolia.search.endpoint.internal
+
+import com.algolia.search.configuration.CallType
+import com.algolia.search.endpoint.EndpointPersonalization
+import com.algolia.search.model.insights.UserToken
+import com.algolia.search.model.personalization.PersonalizationProfileResponse
+import com.algolia.search.model.personalization.PersonalizationStrategy
+import com.algolia.search.model.personalization.SetPersonalizationStrategyResponse
+import com.algolia.search.serialize.RoutePersonalization
+import com.algolia.search.serialize.RouteProfiles
+import com.algolia.search.serialize.internal.Json
+import com.algolia.search.transport.RequestOptions
+import com.algolia.search.transport.internal.Transport
+import io.ktor.http.HttpMethod
+
+internal class EndpointPersonalizationImpl(
+    private val transport: Transport,
+) : EndpointPersonalization {
+
+    override suspend fun getPersonalizationProfile(
```

Comment:
As I understood, you implemented get/delete personalization profile methods in this commit, these method weren't there in the EndpointRecommendation?

Reply:
Yes, it is part of the initial feature request:
> This PR also documents the GET/DELETE of user profiles.